app/controllers/users/confirmations_controller.rb

```diff
@@ -2,8 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def new
+      @user = User.new
```

Comment:
This is the method I am overriding: https://github.com/plataformatec/devise/blob/master/app/controllers/devise/confirmations_controller.rb#L4

Devise does `self.resource`, which I think is really weird and unexpected in a Rails app. I was looking all over for where it was defined before I realized what they were doing. 

In my ideal world, we would probably remove our dependence on Devise for this confirmation stuff. We are overriding enough of it that it is more confusing than helpful, imo.